Exceptions are not visible in testsuite output

### Describe the bug

I'm running the test suite under maven with `-Pauth-server-quarkus`. 
Exceptions in the server are not visible in the log output.

The server logs are also missing from the testsuite output.

### Version

main

### Expected behavior

When code run by the server throws an exception, lets say `NullPointerException`, I'd like full backtrace to be displayed in the test suite output.

### Actual behavior

The exception from failing test case is shown, but the original exception in the server that caused the test case to fail is not shown.

### How to Reproduce?

Modify server to throw unhandled exception.

### Anything else?

_No response_

It seems to me that the intention was to forward server output to parent but this does not seem effective:

https://github.com/keycloak/keycloak/blob/21f700679f21b71ef89985b835f2d3cf7ac049a0/testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/arquillian/containers/KeycloakQuarkusServerDeployableContainer.java#L159

whatever is produced to stdout is not coming through. I found the server logs, including the backtrace are in `testsuite/integration-arquillian/tests/base/target/surefire-reports/*.dumpstream`, but this seems rather inconvenient. The file looks like:

```
...
# Created at 2022-08-03T18:11:57.061
Corrupted channel by directly writing to native stream in forked JVM 1. Stream 'Caused by: java.lang.NullPointerException'.

# Created at 2022-08-03T18:11:57.061
Corrupted channel by directly writing to native stream in forked JVM 1. Stream '        at java.base/java.util.Objects.requireNonNull(Objects.java:221)'.
...
```

--- a/testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/arquillian/containers/KeycloakQuarkusServerDeployableContainer.java
+++ b/testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/arquillian/containers/KeycloakQuarkusServerDeployableContainer.java
@@ -27,6 +27,7 @@ import java.util.List;
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.atomic.AtomicBoolean;
 
+import org.apache.commons.exec.StreamPumper;
 import org.apache.commons.io.FileUtils;
 import org.apache.commons.lang3.SystemUtils;
 import org.jboss.arquillian.container.spi.client.container.DeployableContainer;
@@ -54,6 +55,7 @@ public class KeycloakQuarkusServerDeployableContainer implements DeployableConta
     private KeycloakQuarkusConfiguration configuration;
     private Process container;
     private static AtomicBoolean restart = new AtomicBoolean();
+    private Thread stdoutForwarderThread;
 
     @Inject
     private Instance<SuiteContext> suiteContext;
@@ -75,6 +77,8 @@ public class KeycloakQuarkusServerDeployableContainer implements DeployableConta
         try {
             importRealm();
             container = startContainer();
+            stdoutForwarderThread = new Thread(new StreamPumper(container.getInputStream(), System.out));
+            stdoutForwarderThread.start();
             waitForReadiness();
         } catch (Exception e) {
             throw new RuntimeException(e);
@@ -156,7 +160,7 @@ public class KeycloakQuarkusServerDeployableContainer implements DeployableConta
     private Process startContainer() throws IOException {
         ProcessBuilder pb = new ProcessBuilder(getProcessCommands());
         File wrkDir = configuration.getProvidersPath().resolve("bin").toFile();
-        ProcessBuilder builder = pb.directory(wrkDir).inheritIO().redirectErrorStream(true);
+        ProcessBuilder builder = pb.directory(wrkDir).redirectErrorStream(true);
 
         String javaOpts = configuration.getJavaOpts();
 
@@ -283,7 +287,7 @@ public class KeycloakQuarkusServerDeployableContainer implements DeployableConta
             } catch (Exception ignore) {
             }
         }
-        
+
         log.infof("Keycloak is ready at %s", contextRoot);
     }
 

